spring-kafka/src/main/java/org/springframework/kafka/core/StreamsBuilderFactoryBean.java

```diff
@@ -63,6 +64,14 @@
 
 	private volatile boolean running;
 
+	/**
+	 * Default constructor.
```

Comment:
That's not enough.

We definitely need to say more in which state the object is created.